lang/terraform/terraform.talon

```diff
@@ -0,0 +1,27 @@
+mode: user.code_terraform
+mode: user.auto_lang
+and code.language: terraform
+-
+tag(): user.code_comment_block_c_like
+tag(): user.code_comment_line
+tag(): user.code_data_bool
+tag(): user.code_data_null
+tag(): user.code_imperative
+tag(): user.code_operators_assignment
+tag(): user.code_operators_lambda
+tag(): user.code_operators_math
+
+state {user.terraform_module_blocks}:
+    user.code_terraform_module_block(user.terraform_module_blocks)
+
+^resource <user.text>:
```

Comment:
Why front-anchored?

Reply:
My intuition was that front-anchoring should lead to less false positives during speech recognition. I copied this from the `funky` declaration. Do you think it's a bad idea here?